lib/lts/index.js

```diff
@@ -101,13 +101,32 @@ function ltsVersions(_time, currentDate) {
     });
 }
 
+function getCurrentLts(ltsList) {
+  let latestLts = '';
+  Object.keys(ltsList).forEach(version => {
+    const versionInfo = ltsList[version];
+    const today = new Date();
+    if (
+      new Date(versionInfo.start_date) <= today &&
+      today <= new Date(versionInfo.end_date) &&
+      new Date(versionInfo.maintenance_start_date) >= today
+    ) {
+      latestLts = versionInfo.versionRange;
+    }
+  });
+  if (latestLts) {
```

Comment:
is the aim for this function to short circuit and return if the `latestLTS` is found?

Reply:
This function returns the latest LTS from the list we have in ember-cli-lts.json or node-lts.json. 
It also helps in throwing an exception when the above JSON file is not updated with the latest LTS.